app/assets/stylesheets/chat.scss

```diff
@@ -1120,20 +1120,48 @@
 
 .message__info__actions {
   display: grid;
-  grid-template-columns: 4fr 1fr;
+  grid-template-columns: 19fr 1fr;
 }
 
 .message__actions {
   display: none;
   justify-content: end;
   -webkit-justify-content: flex-end;
-}
 
-.message__actions span {
-  margin-left: 7px;
-  font-size: 13px;
-  font-weight: bold;
-  cursor: pointer;
+  .ellipsis__menubutton {
+    height: 17px;
+    cursor: pointer;
```

Comment:
```css
    color: var(--theme-color, #0a0a0a)
```
Add this to the icon, as it's invisible in dark themes.
<img width="1152" alt="Screenshot 2020-02-02 at 5 36 29 PM" src="https://user-images.githubusercontent.com/3650216/73607927-8e447d00-45e2-11ea-9c93-4d9614ece36c.png">

Reply:
Similar theming problem in edit and delete options.
<img width="138" alt="Screenshot 2020-02-02 at 5 38 21 PM" src="https://user-images.githubusercontent.com/3650216/73607957-d368af00-45e2-11ea-882e-c240a777fe6c.png">